### PR TITLE
updated dependencies to support Mac ARM binaries

### PR DESCRIPTION
**Pull request by jotka**
updated dependencies to support Mac ARM binaries

### CODE REVIEW

**Comment by ankitm123** [comment]
Running the changes in an existing cluster leads to issues:
```
 Error: Unsupported argument
│ 
│   on .terraform/modules/jx/main.tf line 34, in provider "kubernetes":
│   34:   load_config_file = false
│ 
│ An argument named "load_config_file" is not expected here.
╵
╷
│ Error: Unsupported argument
│ 
│   on .terraform/modules/jx/main.tf line 51, in provider "helm":
│   51:     load_config_file = false
│ 
│ An argument named "load_config_file" is not expected here.
```

File: .terraform.lock.hcl [hunk 1]
```diff
@@ -1,158 +0,0 @@
-# This file is maintained automatically by "terraform init".
```
Comment (ankitm123) [inline .terraform.lock.hcl:1]: Why was this deleted? You can run terraform init -upgrade to update it? No strong opinions in keeping this file ... but a lock file will gurantee the dependency versions

Comment (jotka) [reply .terraform.lock.hcl:1]: I mean, the lock file should not be in the git repo, as it will be regenerated always with the dependencies from main.tf. It's often added to .gitignore.

Comment (jotka) [reply .terraform.lock.hcl:1]: BTW i've installed jx on GKE using MacBook Pro 16 ARM :)

Comment (ankitm123) [reply .terraform.lock.hcl:1]: https://support.hashicorp.com/hc/en-us/articles/4610780468627-Terraform-lock-file-management -> I think either is fine, I am ok removing it ...

Comment (jotka) [reply .terraform.lock.hcl:1]: hey @ankitm123 will you approve / merge? :)

Comment (ankitm123) [reply .terraform.lock.hcl:1]: https://github.com/jenkins-x/terraform-google-jx/pull/223#issuecomment-1166315497

Comment (jotka) [reply .terraform.lock.hcl:1]: fixed.

**Comment by ankitm123** [comment]
Tested on ur jx GCP infra:
```
No changes. Your infrastructure matches the configuration.
```

**Review by ankitm123 — APPROVED** [review]
